hashintel/hash
# Add ability to upgrade the version of expected entity types in links

## 🌟 What is the purpose of this PR?



This PR adds the ability to upgrade the version of expected entity types in links in the entity type editor.

## 🔗 Related links





- [Asana task](https://app.asana.com/0/1203358502199087/1204080201525406/f) _(internal)_


## Pre-Merge Checklist 🚀

### 🚢 Has this modified a publishable library?




This PR:












### 📜 Does this require a change to the docs?




The changes in this PR:






### 🕸️ Does this require a change to the Turbo Graph?




The changes in this PR:







## ❓ How to test this?



1.  Create two entity types;
2.  Create a link in the first entity type and add the second entity type as an expected entity type;
3.  Make changes to the second entity type and publish it to create a new version;
4. Go back to the first entity type (make sure you reload the page) and check if you can upgrade the version of the expected entity type in the link.

## 📹 Demo



![demo](https://github.com/hashintel/hash/assets/37453800/7c65f4b1-3bf3-40af-a9df-e6b940044204)
